docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -312,67 +151,30 @@ The following attributes are available for `esp32` boards:
 | ---- | ---- | --------- | ----------- |
 | `pins` | object | Required | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of any GPIO pins you wish to use as input/output with the [`GPIOPin` API](/program/apis/#gpio-pins). |
 | `analogs` | object | Optional | Attributes of any pins that can be used as analog-to-digital converter (ADC) inputs. See [configuration info](/components/board/#analogs). |
-<!-- | `webhook` | string | Optional | A webhook's URL. See [GitHub](viam-labs/webhook-template) for an example. |
-| `webhook-secret` | string | Optional | The robot secret for a configured `webhook`. See [GitHub](viam-labs/webhook-template) for an example. | -->
-
-### Generate a New Project from the Micro-RDK Template
-
-[Start Docker](https://docs.docker.com/config/daemon/start/) on your development machine.
-If you haven't already, complete Docker's [Linux post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to set up Docker to run whenever your system boots up.
-
-Use [the Micro-RDK template](https://github.com/viamrobotics/micro-rdk-robot-template.git) to create a new Micro-RDK project to upload to your ESP32 by running:
-
-```sh { class="command-line" data-prompt="$"}
-cargo generate --git https://github.com/viamrobotics/micro-rdk-robot-template.git
-```
 
-You will be prompted to paste your robot's JSON configuration into the terminal.
-To obtain this:
-
-- Navigate to [your new robot's](/installation/prepare/microcontrollers/#create-a-new-robot) page on [the Viam app](https://app.viam.com).
-- Click on the **Setup** tab.
-  Keep your `Mode` and `Architecture` selections at default.
-- Click the **Copy viam-server config** button on the right side of the **Setup** tab.
-  The micro-RDK needs this JSON file, which contains your robot part secret key and cloud app address, to connect to the [Viam app](https://app.viam.com).
-- Paste the `viam-server` config into your terminal when prompted.
-
-{{% snippet "secret-share.md" %}}
-
-### Upload the Micro-RDK to your ESP32
-
-Now, upload the project to connect to your ESP32 and remotely control it live on [the Viam app](https://app.viam.com):
-
-{{< tabs >}}
-{{% tab name="Use Canon" %}}
-
-```sh { class="command-line" data-prompt="$"}
-cd <your-path-to/your-project-directory>
-canon bash -lc "make build-esp32-bin"
-make flash-esp32-bin
-```
+### Troubleshooting
 
-{{% /tab %}}
-{{% tab name="Local environment" %}}
+#### Linux Port Permissions
 
-Make sure you have run `. ~/dev/esp/export-rs.sh` and `. ~/dev/esp/esp-idf/export.sh` before running the following command:
+If a "Permission Denied" or similar port error occurs, first check the connection of the ESP32 to the machine's USB port.
+If connected and the error persists, run `sudo usermod -a -G dialout $USER` to add the current user to the `dialout` group, then try again.
```

Comment:
Add that the user may need to restart the terminal before trying again

Reply:
done!